application/__init__.py

```diff
@@ -0,0 +1,37 @@
+from flask import Flask, jsonify
+from flask_restful import Api
+from werkzeug.exceptions import HTTPException
+from werkzeug.exceptions import default_exceptions
+from flask_sqlalchemy import SQLAlchemy
+from application.config import app_config
+
+db = SQLAlchemy()
+
+def create_app(config_name):
+    app = Flask(__name__)
+
+    @app.errorhandler(Exception)
+    def handle_error(e):
+        code = 500
+        if isinstance(e, HTTPException):
+            code = e.code
+        return jsonify(error=str(e)), code
+
+    for ex in default_exceptions:
+        app.register_error_handler(ex, handle_error)
+
+    app.config.from_object(app_config[config_name])
+    app.config.from_pyfile('config.py')
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+    db.init_app(app)
+
+    api = Api(app)
+    api.prefix = '/api'
```

Comment:
How does this prefix work??

Reply:
@Chulstro I'd also like to know this. Maybe we could add an issue card with a research tag to share what we find out about this.